v2/tests/log_context_property_basic_types_ut/log_context_property_basic_types_ut.c

```diff
@@ -272,35 +272,35 @@ static void int32_t_copy_succeeds(void)
 
 /* Tests_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_007: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).copy shall copy the bytes of the int64_t value from the address pointed by src_value to dst_value. ]*/
 /* Tests_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_008: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).copy shall succeed and return 0. ]*/
-static void int32_t_copy_succeeds_2(void)
+static void int64_t_copy_succeeds_2(void)
 {
     // arrange
-    int32_t src = INT32_MAX;
-    int32_t dst = 43;
+    int64_t src = INT64_MAX;
+    int64_t dst = 43;
 
     setup_mocks();
 
     // act
-    int result = LOG_CONTEXT_PROPERTY_TYPE_IMPL(int32_t).copy(&dst, &src);
+    int result = LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).copy(&dst, &src);
 
     // assert
     POOR_MANS_ASSERT(result == 0);
-    POOR_MANS_ASSERT(dst == INT32_MAX);
+    POOR_MANS_ASSERT(dst == INT64_MAX);
     POOR_MANS_ASSERT(actual_and_expected_match);
 }
 
 /* LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).free */
 
 /* Tests_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_009: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).free shall return. ]*/
-static void int32_t_free_returns(void)
+static void int64_t_free_returns(void)
 {
     // arrange
-    int32_t value = INT32_MAX;
+    int64_t value = INT64_MAX;
 
     setup_mocks();
 
     // act
-    LOG_CONTEXT_PROPERTY_TYPE_IMPL(int32_t).free(&value);
+    LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).free(&value);
```

Comment:
>free(&value) [](http://example.com/codeflow?start=44&length=12)

No linux nerdy compiler sees this as a warning? freeing the address of a stack variable... well... through some layers of function pointers... so maybe not.

Reply:
The free for these basic types do nothing actually.
When we'd have a ref counted type for example (I secretly hope that we'll have custom types at some point), then the free for that one will have to do something.